types/src/transaction/mod.rs

```diff
@@ -270,6 +339,7 @@ impl RawTransaction {
         format!(
             "RawTransaction {{ \n\
              \tsender: {}, \n\
+             \tsecondary_signers: {:?}, \n\
```

Comment:
Nit: probably don't want to use the debug print here as (IIRC) it will print the addresses in a different (not very human friendly) format.